community-operators/namespace-configuration-operator/1.0.7/manifests/namespace-configuration-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,329 @@
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "NamespaceConfig",
+          "metadata": {
+            "name": "test-namespaceconfig"
+          },
+          "spec": {
+            "labelSelector": {
+              "matchLabels": {
+                "type": "secure"
+              }
+            },
+            "templates": [
+              {
+                "objectTemplate": "apiVersion: v1\nkind: ResourceQuota\nmetadata:\n  name: standard-sandbox\n  namespace: {{ .Name }}\nspec:\n  hard:\n    pods: \"4\" \n    requests.cpu: \"1\" \n    requests.memory: 1Gi \n    requests.ephemeral-storage: 2Gi \n    limits.cpu: \"2\" \n    limits.memory: 2Gi \n    limits.ephemeral-storage: 4Gi\n"
+              },
+              {
+                "objectTemplate": "kind: EgressNetworkPolicy\napiVersion: network.openshift.io/v1\nmetadata:\n  name: air-gapped\n  namespace: {{ .Name }}  \nspec:\n  egress:\n  - type: Deny\n    to:\n      cidrSelector: 0.0.0.0/0\n"
+              }
+            ]
+          }
+        },
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "UserConfig",
+          "metadata": {
+            "name": "test-user-config"
+          },
+          "spec": {
+            "providerName": "my-provider",
+            "templates": [
+              {
+                "objectTemplate": "apiVersion: v1\nkind: Namespace\nmetadata:\n  name: {{ .Name }}-sandbox\n"
+              },
+              {
+                "objectTemplate": "apiVersion: v1\nkind: ResourceQuota\nmetadata:\n  name: standard-sandbox\n  namespace: {{ .Name }}-sandbox\nspec:\n  hard:\n    pods: \"4\" \n    requests.cpu: \"1\" \n    requests.memory: 1Gi \n    requests.ephemeral-storage: 2Gi \n    limits.cpu: \"2\" \n    limits.memory: 2Gi \n    limits.ephemeral-storage: 4Gi\n"
+              }
+            ]
+          }
+        }
+      ]
+    capabilities: Full Lifecycle
+    categories: Security
+    certified: "false"
+    containerImage: quay.io/redhat-cop/namespace-configuration-operator
+    createdAt: 5/28/2019
+    description: This operator provides a facility to define and enforce namespace configurations
+    operatorframework.io/suggested-namespace: namespace-configuration-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.4.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/redhat-cop/namespace-configuration-operator
+    support: Best Effort
```

Comment:
```suggestion
    support: Best Effort
    olm.properties: |-
        [
            {
                "type": "olm.maxOpenShiftVersion",
                "value": "4.8"
            }
        ]
```

Reply:
This bundle 1.0.7 is not using deprecated/removed APIs in 1.22. So, we should not add that here. 
The errors are faced by checking bundle 0.0.1 see: https://github.com/operator-framework/community-operators/pull/4142/checks?check_run_id=2904751610#step:3:490

The error/warning msgs returns the name of the bundle always to let us know the what is the bundle which the issue was faced in this case see: ` msg="Error: Value : (namespace-configuration-operator.v0.0.1)`